src/style.css

```diff
@@ -1,45 +1,75 @@
+/* Basic styling */
 * {margin: 0; padding: 0; border: 0; vertical-align: baseline;}
 html {box-sizing: border-box;}
 *, *:before, *:after {box-sizing: inherit;}
-
-html, body {
-    height: 100%;
-    margin: 0;
-  }
+html, body {height: 100%; margin: 0;}
 
 body {
   font-family: "Segoe UI", sans-serif;
   background: #1A2933;
   color: #FFF;
 }
+
+h1 {
+  margin: 0 0 10px 0;
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+p {
+  margin-top: 10px;
+  color: rgba(255,255,255,0.4);
+}
+
+/* Styling of window frame and titlebar */
 body {
   border: 1px solid #48545c;
   overflow-y: hidden;
 }
 
-
+body.maximized {
+  border: none;
+}
 
 #titlebar {
   display: block;
   position: fixed;
   height: 32px;
   width: calc(100% - 2px);
 }
+
+.maximized #titlebar {
+  width: 100%;
+  padding: 0;
+}
+
+#main {
+  position: absolute;
+  height: calc(100% - 32px);
+  top: 32px;
```

Comment:
This doesn't take into account the 1px border at the top, so it's not far down enough.

![Untitled-9](https://user-images.githubusercontent.com/14579418/68144143-e956cd00-ff2a-11e9-8f19-87a8556b91cd.png)


I'm reverting it to `margin-top: 32px`

Reply:
Oh yeah sorry. I should have realized that it wouldn't take into account the border with `position: absolute`. But I did that for a reason, because something else that I changed messed that up.

Reply:
Oh I did it this way because when the 1px border was removed, the margin messed up the title bar, shifting it 32px down instead.
![image](https://user-images.githubusercontent.com/20509848/68166163-97a75480-ffb5-11e9-8fea-9d5bafdaea16.png)
But since you don't want the border removed I guess it doesn't matter.

Reply:
Fixing up the absolute positioning of `#main` isn't hard though, I've done it that way in my own app